### ISSUE TITLE
There *may* be many images per article displayed in the pop-ups

### ISSUE THREAD

**Issue by wu-lee** [TransitionbyDesign/homemaker#28]

**Comment by wu-lee** [comment]
This is currently possibly simply by inserting a link in the mark-down via the forestry.io UI.  The image can be an URL (in which case it needs to have been uploaded somewhere public, and doesn't get copied into the code repository), or it can be selected from the current library of assets (so it needs to have been previously uploaded into the `content/assets` folder in the code repository, which can be done via the forestry.io UI)

(Note, forestry.io can also put assets like this on some 3rd party storage like Amazon S3.)

This is in addition to a "hero image" which serves as a focus for the page and a thumbnail when listing blog posts - a field inherited in the original forestry.io starter plug-in I am building on. 

We need to decide if we want to keep this hero image field in all cases, which depends on whether we want thumbnails, and whether we want a large image at the top of all articles.

Especially so in the case of video content. If we do want thumbnails, I'm not sure how easily we can transform an arbitrary uploaded video file into a thumbnail image in general. It's technically feasible, but might not be easy to guarantee that it picks a representative frame, and also I'm not sure if there are off-the-shelf plug-ins which allow me to do this without spending potentially days building something new to do that. So if we want thumbnails displayed somewhere for videos, we might do well just to manually provide a thumbnail via the hero_image field.


I see there's a Gatsby plug-in to access Notion.so data (in fact, several, some of which are clearly unofficial since they've reverse-engineered the unpublished API). This may also be an option, but then again, we don't have a lot of time left to look into it.

**Comment by wu-lee** [comment]
As mentioned above, images can be inserted into the documents using the rich text buttons in the forestry.io UI

So I think this is dealt with? Moving to review pile.

**Comment by Charlie-TbD** [comment]
Yes, although not in the way we agreed in the October project meeting where we asked for a gallery function (e.g. multiple images in the place the hero image is with arrows to cycle through key images). Will move to done.
